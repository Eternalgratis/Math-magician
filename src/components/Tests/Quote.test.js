import Renderer from 'react-test-renderer';
import Quote from '../Quote';

it('Renders Quote component', () => {
  const tree = Renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});
