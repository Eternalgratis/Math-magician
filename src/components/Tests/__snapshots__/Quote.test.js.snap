// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Home component 1`] = `
<div
  className="quote"
>
  <p>
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. - William Paul Thurston
  </p>
</div>
`;

exports[`Renders Quote component 1`] = `
<div
  className="quote"
>
  <p>
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. - William Paul Thurston
  </p>
</div>
`;

exports[`Renders quote component 1`] = `
<div
  className="quote"
>
  <p>
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. - William Paul Thurston
  </p>
</div>
`;
